python/test/test_property_graph.py

```diff
@@ -16,6 +16,23 @@ def test_load(graph):
     assert len(graph.loaded_edge_schema()) == 3
 
 
+def test_schemas(graph):
+    assert len(graph.loaded_node_schema()) == 17
```

Comment:
why do `loaded` start with 17; should it be 0 initially (or maybe something is happening when before the graph comes here as the argument, which is the drawback of reviewing code without context :).

Reply:
The test graph has all properties loaded by default. I think mostly for historical reasons, but it doesn't seem like a big issue to me.

Reply:
That is totally fine.  If somebody updates that eventually this test will fail so they will have to fix, so all is good.